creator-node/src/app.js

```diff
@@ -63,6 +40,76 @@ app.use(errorHandler)
  * @param {ServiceRegistry} serviceRegistry object housing all Content Node Services
  */
 const initializeApp = (port, serviceRegistry) => {
+  const app = express()
+
+  // middleware functions will be run in order they are added to the app below
+  //  - loggingMiddleware must be first to ensure proper error handling
+  app.use(loggingMiddleware)
+  app.use(bodyParser.json({ limit: '1mb' }))
+  app.use(readOnlyMiddleware)
+  app.use(cors())
+
+  // Rate limit routes
+  app.use('/users/', userReqLimiter)
+  app.use('/track*', trackReqLimiter)
+  app.use('/audius_user/', audiusUserReqLimiter)
+  app.use('/metadata', metadataReqLimiter)
+  app.use('/image_upload', imageReqLimiter)
+  app.use('/ursm_request_for_signature', URSMRequestForSignatureReqLimiter)
+  app.use('/batch_cids_exist', batchCidsExistReqLimiter)
+  app.use('/batch_image_cids_exist', batchCidsExistReqLimiter)
+  app.use(getRateLimiterMiddleware())
+
+  const prometheusRegistry = serviceRegistry.prometheusRegistry
+
+  // Metric tracking middleware
+  app.use(
+    prometheusMiddleware({
+      // Use existing registry for compatibility with custom metrics. Can see
+      // the metrics on /prometheus_metrics
+      promRegistry: prometheusRegistry.registry,
+      // Override metric name to include namespace prefix
+      httpDurationMetricName: `${prometheusRegistry.namespacePrefix}_http_request_duration_seconds`,
+      // Include HTTP method in duration tracking
+      includeMethod: true,
+      // Include HTTP status code in duration tracking
+      includePath: true,
+      // Disable default gauge counter to indicate if this middleware is running
+      includeUp: false,
+      // The buckets in seconds to measure requests
+      buckets: [0.2, 0.5, ...exponentialBucketsRange(1, 60, 8)],
```

Comment:
seems we can simplify this to `...exponentialBucketsRange(0.2, 60, 10)` for effectively same results?
but also don't think we need 10 buckets since we always have the average anyway - how about `(0.1, 60, 5)`

Reply:
issue with fn so thats why the min is not a decimal. resolving